packages/services/src/view-service/fees.test.ts

```diff
@@ -92,48 +70,6 @@ describe('extractAltFee', () => {
     expect(result.equals(swapAssetId)).toBeTruthy();
   });
 
-  it('extracts the fee from dutchAuctionScheduleActions', () => {
```

Comment:
question: why are the tests removed?

Reply:
dutch auctions for multi-fees are more complex than I thought, and will be handled in a separate PR. As such, the unit tests should be removed since the corresponding logic in the `extractAltFees` method was removed. We can fold the unit tests back in during https://github.com/penumbra-zone/web/issues/1414

Reply:
in this case, it would probably be better to use the `it.skip()` along with the `todo: fix tests` comment surely not forget about them

Reply:
sure good suggestion 👍